src/components/shared/Buttons.js

```diff
@@ -11,9 +11,10 @@ export const ButtonBasic = styled.button`
   border: 0;
 `
 
-export function ButtonSide({ action }) {
+export function ButtonSide({ action, learnMore = false }) {
```

Comment:
can we make this a string instead of a bool. That way we could have different text in the future for different uses of `ButtonSide`

Reply:
Yea, we could take button text

Reply:
The purpose of this is to check whether to show the message or not. I'll add another parameter to allow for custom mesages.